CONTRIBUTING.md

```diff
@@ -5,6 +5,58 @@ GitHub pull requests.  This document outlines some of the conventions on
 development workflow, commit message formatting, contact points and other
 resources to make it easier to get your contribution accepted.
 
+# Issues
+
+When filing an issue, make sure to answer these questions in the issue body:
+
+- Environment
+  1. rkt: Paste the output of `rkt version`.
+  2. Linux kernel: Paste the output of `uname -srm`
+  3. Distribution/Operating System incl. version
```

Comment:
Only on distros conforming with https://www.freedesktop.org/software/systemd/man/os-release.html but as example good for most cases - quite useful.